juriscraper/pacer/reports.py

```diff
@@ -200,6 +200,20 @@ def download_pdf(
             # Add parameters to the PACER base url and make a GET request
             req_timeout = (60, 300)
             r = requests.get(url, params=params, timeout=req_timeout)
+
+            # If the response is an HTML document and it doesn't contain an
+            # IFRAME, the magic link document is no longer available
+            error = None
+            if is_html(r) and not "iframe" in r.text:
+                error = (
+                    f"Magic link document not available in case: "
+                    f"URL: {url}, caseid: {pacer_case_id}, "
+                    f"magic_num: {pacer_magic_num}"
```

Comment:
I'd switch these variables around so the URL comes last. Makes things easier to read since it's so long.

Reply:
yes, this is better, changed!